### ISSUE TITLE
Failed to build the package

### ISSUE THREAD

**Issue by LeungHEX** [gaowenliang/imu_utils#12]
Once  follow the constrction in the readme, but the bulid console show the error msg like :+1: 

- Could not find a package configuration file provided by "code_utils" with
- any of the following names:
- 
- code_utilsConfig.cmake
- code_utils-config.cmake

then i try to build the code_utils without imu_utils it shows the following error:

- fatal error: backward.hpp: No such file or directory

I am pretty sure that the files pull from your git is correct and complete, but why it cannot build?

Or can someone else help me?

**Comment by LeungHEX** [comment]
Besides, I copy the backward.hpp to where the cpp need, it built sucessfully. But I have to build code_utils first then add the imu_utils into the workspace, it seems that this process is not the same as you introduced in the readme, is there something wrong in my env? Thanks in advance if there is any useful tips.

**Comment by Sayheyheyhey** [comment]
The fatal error I faced is
XXXX/code_utils/src/sumpixel_test.cpp:2:24: fatal error: backward.hpp: No such file or directory

You can edit the "sumpixel_test.cpp" to solve this problem, 
change the  
#include "backward.hpp"
to  
#include "code_utils/backward.hpp"

**Comment by jsYangCode** [comment]
@LeungHEX , hi, my friend,  i also meet the problem as yours, have you solved it? would you please give me some help? thank U.

**Comment by Sayheyheyhey** [comment]
像上一条评论里写的一样
只需要把sumpixel_test.cpp文件中的下边这句
#include "backward.hpp"
改成这句
#include "code_utils/backward.hpp"
就解决了

**Comment by jsYangCode** [comment]
@Sayheyheyhey ok 了 谢谢您

**Comment by luzhaozhang2276** [comment]
@Sayheyheyhey  Using your method, the problem is still unresolved.
I compile code_utils first, then add imu_utils in, compile again, and it succeeds.

**Comment by beduffy** [comment]
Yep, compiling with code_utils only in the src folder worked (after fixing some dependencies e.g. backward.hpp, libf77blas.so, libspqr.so) and then I moved imu_utils to the same folder and ran catkin_make again. This should be documented or fixed for other people.

**Comment by Hezip** [comment]
This is a problem with the **compile chain**. 
You need to add such a sentence in the **package.yaml** of **imu_utils** 
**<buildtool_depend> code_utils </ buildtool_depend>**

**Comment by Tianci-Wen** [comment]
> Yep, compiling with code_utils only in the src folder worked (after fixing some dependencies e.g. backward.hpp, libf77blas.so, libspqr.so) and then I moved imu_utils to the same folder and ran catkin_make again. This should be documented or fixed for other people.

I solved the problems by this way.
